src/main/java/duke/Todos.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+/**
+ * A task type that the chatting bot can create.
+ */
+public class Todos extends Task {
```

Comment:
Maybe you could rename the class to `ToDo` (as plurals would lead people to think of a collection of objects)?